phpstan-baseline.neon

```diff
@@ -42,6 +42,12 @@ parameters:
 			count: 1
 			path: src/Monolog/Formatter/WildfireFormatter.php
 
+		-
+			message: '#^Method Monolog\\Handler\\BufferHandler\:\:setHandler\(\) has no return type specified\.$#'
```

Comment:
baselined this error, as adding a new return type is likely a BC break

Reply:
this method [was recently added](https://github.com/Seldaek/monolog/pull/1946) and not yet part of a release -> I added the return type instead